lib/ddg-atb.js

```diff
@@ -16,7 +16,8 @@
 
 "use strict";
 
-const {Cu} = require("chrome");
+const {Cu, CC} = require("chrome");
+const XMLHttpRequest = CC("@mozilla.org/xmlextras/xmlhttprequest;1");
```

Comment:
How necessary do you think it is to switch? The nice thing about using XLMHttpRequest is that we can copy the same function from the chrome extension.